File: swing/src/main/java/wannabe/swing/renderer/ThreeDSquare.java
package wannabe.swing.renderer;

import java.awt.Graphics;
import wannabe.Projected;

public class ThreeDSquare extends SwingRenderer {
  @Override public void draw(Graphics g, Projected r) {
    g.draw3DRect(r.left, r.top, r.size, r.size, true);
  }
}

File: swing/src/main/java/wannabe/swing/renderer/ThreeDSquare.kt
package wannabe.swing.renderer

import wannabe.swing.SwingProjected
import java.awt.Graphics

class ThreeDSquare : SwingRenderer() {
  override fun draw(
    g: Graphics,
    p: SwingProjected
  ) {
    g.draw3DRect(p.left, p.top, p.size, p.size, true)
  }
}
